core/src/trezor/ui/model/common.py

```diff
@@ -0,0 +1,136 @@
+from micropython import const
+
+from trezor import ui, utils
+
+if False:
+    from typing import List, Union
+
+    TextContent = Union[str, int]
+
+# needs to be different from all colors and font ids
+BR = const(-256)
+BR_HALF = const(-257)
+
+
+# FIXME: ugly & had to get rid of const()
+if utils.MODEL == "1":
+    TEXT_HEADER_HEIGHT = 13
```

Comment:
i'm wondering if this should be exported either from C, or somehow otherwise configured at compile-time